integration-cli/docker_cli_help_test.go

```diff
@@ -100,5 +87,9 @@ func TestCmdHelpWidth(t *testing.T) {
 		}
 	}
 
+	if count < 10 {
```

Comment:
Not sure how to get it.  All I have is the count of indented lines - which should be the right number but if something goes wrong with "docker help" and instead I get something else I don't really have a good way to know it.